Source/UnitTests/CMakeLists.txt

```diff
@@ -1,5 +1,14 @@
 enable_testing()
 add_custom_target(unittests)
+
+if(APPLE)
+  set(EXECUTABLE_PATH build/arm64/Source/UnitTests)
+else()
+  set(EXECUTABLE_PATH build/Source/UnitTests)
+endif()
+add_custom_command(TARGET unittests POST_BUILD
+  COMMAND ${CMAKE_COMMAND} -E copy_directory "${CMAKE_SOURCE_DIR}/Data/Sys" "${CMAKE_SOURCE_DIR}/${EXECUTABLE_PATH}/Sys"
```

Comment:
I'm... not sure what you're trying to do here. What is `EXECUTABLE_PATH` and why are you so sure of its relationship to the source directory? Do you just want `CMAKE_BINARY_DIR` or what...?

Reply:
In fact, do you just want:

```
add_custom_command(TARGET unittests POST_BUILD
  COMMAND ${CMAKE_COMMAND} -E copy_directory "${CMAKE_SOURCE_DIR}/Data/Sys" "${CMAKE_CURRENT_BINARY_DIR}/Sys"
)
```

?